change/@microsoft-fast-tooling-f25a09d2-ac4d-45db-827c-be83b473301f.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Added Playwright and first test",
+  "packageName": "@microsoft/fast-tooling",
+  "email": "44823142+williamw2@users.noreply.github.com",
+  "dependentChangeType": "patch"
```

As these are all testing infrastructure that doesn't touch exported code, we don't need to bump.

```suggestion
  "type": "none",
  "comment": "Added Playwright and first test",
  "packageName": "@microsoft/fast-tooling",
  "email": "44823142+williamw2@users.noreply.github.com",
  "dependentChangeType": "none"
```